test/main_unit_test_bmi.c

```diff
@@ -24,15 +18,13 @@ main(int argc, const char *argv[]){
     Bmi *model = (Bmi *) malloc(sizeof(Bmi));
 
     // Register BMI model
-    /* TODO: check if this bmi function needs to be model specific?
-            or can be called 'regester_bmi()'?*/
     printf(" registering BMI model...\n");
     register_bmi_topmodel(model);
 
     // Test BMI: CONTROL FUNCTION initialize()
     {
         printf(" initializing...");
-        const char *cfg_file = argv[1];
+        const char *cfg_file = "./data/topmod_unit_test.run";
```

Comment:
Is this intended to be hard coded?

Reply:
yah, unit test are cut & dry.  This is just easier.